Source/com/drew/imaging/heif/HeifMetadataReader.java

```diff
@@ -23,7 +23,6 @@
 import com.drew.lang.annotations.NotNull;
 import com.drew.metadata.Metadata;
 import com.drew.metadata.heif.HeifBoxHandler;
-
```

Comment:
Ensure that no unwanted changes are pushed.

Reply:
These methods are required to fetch exif data bytes